core/src/main/scala/org/apache/spark/storage/BlockManager.scala

```diff
@@ -1241,6 +1241,12 @@ private[spark] class BlockManager(
     futureExecutionContext.shutdownNow()
     logInfo("BlockManager stopped")
   }
+
+  private[storage] def hasCachedBlocks: Boolean = {
```

Comment:
Please add a java doc here to explain what this is used for. Right now this doesn't count broadcast blocks, and we should explain why.